spaceship/lib/spaceship/connect_api/tunes/tunes.rb

```diff
@@ -200,6 +200,107 @@ def patch_app(app_id: nil, attributes: {}, app_price_tier_id: nil, territory_ids
           tunes_request_client.patch("apps/#{app_id}", body)
         end
 
+        #
+        # appDataUsage
+        #
+
+        def get_app_data_usages(app_id: nil, filter: {}, includes: nil, limit: nil, sort: nil)
```

Comment:
It looks like `app_id` is required here and should probably not be `nil`. Same for other methods added to this file. Maybe add some validation? I'm not sure if there's a way the `app_id` could end up being `nil` here, but I assume if it somehow is users will get weird errors from the App Store Connect API